server/index.js

```diff
@@ -1,42 +1,47 @@
-import http from 'http'
 import { resolve, join } from 'path'
 import { parse } from 'url'
+import http from 'http'
 import send from 'send'
+import accepts from 'accepts'
+import {
+  renderToHTML,
+  renderErrorToHTML,
+  renderJSON,
+  renderErrorJSON,
+  sendHTML
+} from './render'
 import Router from './router'
-import { render, renderJSON, errorToJSON } from './render'
 import HotReloader from './hot-reloader'
 import { resolveFromList } from './resolve'
 
 export default class Server {
-  constructor ({ dir = '.', dev = false, hotReload = false }) {
```

Comment:
I really like this.
I don't think there's a much of a use case for `dev: false` and `hotReload: true`

We may need to enable hotReload in production for mobile apps, but we could do it later.